src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+        boolean isEqualToLIST_PRINT_COMMAND = userInputSentence.toLowerCase().equals(Commands.LIST_PRINT_COMMAND);
+        boolean isEqualToSET_TASK_DONE_COMMAND = userInputSentence.length() >= Commands.SET_TASK_DONE_COMMAND.length() &&
+                    userInputSentence.substring(0, Commands.SET_TASK_DONE_COMMAND.length()).equals(Commands.SET_TASK_DONE_COMMAND);
+
+        if (isEqualToTERMIATION_SIGNAL){
+            exit();
+        } else if (isEqualToLIST_PRINT_COMMAND) {
+            printTaskList();
+        } else if (isEqualToSET_TASK_DONE_COMMAND) {
+            String[] splitCommand = userInputSentence.split(" ");
+            Integer setTaskId = Integer.valueOf(splitCommand[1]);
+
+            setTaskDone(setTaskId - 1); // -1 because array's index starts from 0
+        } else {
+            executeTaskAddingCommand(userInputSentence);
+        }
+    }
+
+    private static void executeTaskAddingCommand(String userInputSentence) {
+        boolean isAddTodo = userInputSentence.length() >= Commands.ADD_TODO_COMMAND.length() &&
+                userInputSentence.substring(0, Commands.ADD_TODO_COMMAND.length()).equals(Commands.ADD_TODO_COMMAND);
+        boolean isAddDeadline = userInputSentence.length() >= Commands.ADD_DEADLINE_COMMAND.length() &&
+                userInputSentence.substring(0, Commands.ADD_DEADLINE_COMMAND.length()).equals(Commands.ADD_DEADLINE_COMMAND);
+        boolean isAddEvent = userInputSentence.length() >= Commands.ADD_EVENT_COMMAND.length() &&
+                userInputSentence.substring(0, Commands.ADD_EVENT_COMMAND.length()).equals(Commands.ADD_EVENT_COMMAND);
+
+        TaskType taskTypeToAdd = TaskType.TASK;
+        if (isAddTodo) {
+            taskTypeToAdd = TaskType.TODO;
+        } else if (isAddDeadline) {
+            taskTypeToAdd = TaskType.DEADLINE;
+        } else if (isAddEvent) {
+            taskTypeToAdd = TaskType.EVENT;
+        }
+
+        switch (taskTypeToAdd) {
+        case TODO: {
+            String description = userInputSentence.substring(Commands.ADD_TODO_COMMAND.length());
+            tasks[inputStoredIndex] = new Todo(description);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
+        }
+        case DEADLINE: {
+            String[] splitCommands = userInputSentence.substring(Commands.ADD_DEADLINE_COMMAND.length()).split("/by ");
+            tasks[inputStoredIndex] = new Deadline(splitCommands[0], splitCommands[1]);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
+        }
+        case EVENT: {
+            String[] splitCommands = userInputSentence.substring(Commands.ADD_EVENT_COMMAND.length()).split("/at ");
+            tasks[inputStoredIndex] = new Event(splitCommands[0],splitCommands[1]);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
```

Comment:
Suggestion would to be have a method for the addition of each task type rather than to handle it in case statements